build/mac/BUILD.gn

```diff
@@ -568,3 +585,17 @@ copy("copy_dmg_pkg") {
 
   outputs = [ "$root_out_dir/{{source_file_part}}" ]
 }
+
+if (brave_enable_updater) {
+  crx3("crx") {
+    inputs = [ target_dmg_path ]
+    output = "$root_out_dir/$chrome_product_full_name.crx3"
+    base_dir = get_path_info(target_dmg_path, "dir")
+
+    # The private key file is not in Git for obvious reasons. It needs to be
+    # copied into this directory manually. Changing the private key requires
+    # also changing kBravePublisherKeyHash in crx_verifier.cc.
+    key = "//brave/updater/crx-private-key.der"
```

Comment:
sec-review is required here. Not sure if it's right to put the private key during the build at this location.

Reply:
cc @brave/sec-team. Would you please take a look?

Reply:
I requested a review by opening an issue at https://github.com/brave/reviews/issues/1562.

Reply:
conclusions:

* this key should be regenerated instead of using the existing crx signing key
* fine to save it as a jenkins credential, load it into this directory for the build, then remove it immediately afterwards

Reply:
Regenerated the key in b1362a7c4473b95b136ebf74467e44d07995b47e.